src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -162,6 +173,11 @@ export class SchoolSelectField extends React.Component {
             <LoadingIndicator message={`Searching ${institutionQuery}...`}/>
           </div>
           }
+          {showNoResultsFound && <div className="no-results-box">
```

Comment:
I prefer a flat props interface on views- easier to understand exactly what is controlled by the props at the expense of being a little more verbose. Putting the interpretation into the reducer adds some indirection but also makes the logic very easy to unit test.

Reply:
Thanks for explaining, we should standardize this approach.